http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/api/RuntimeRoutes.scala

```diff
@@ -0,0 +1,65 @@
+package org.broadinstitute.dsde.workbench.leonardo
+package http
+package api
+
+import _root_.java.time.Instant
+import java.util.UUID
+import java.util.concurrent.TimeUnit
+
+import akka.http.scaladsl.marshalling.ToResponseMarshallable
+import akka.http.scaladsl.model.StatusCodes
+import akka.http.scaladsl.server
+import akka.http.scaladsl.server.Directives._
+import cats.effect.{IO, Timer}
+import cats.implicits._
+import cats.mtl.ApplicativeAsk
+import org.broadinstitute.dsde.workbench.leonardo.http.api.LeoRoutes.validateRuntimeNameDirective
+import org.broadinstitute.dsde.workbench.leonardo.http.service.RuntimeService
+import org.broadinstitute.dsde.workbench.leonardo.http.service.RuntimeService.CreateRuntimeRequest
+import org.broadinstitute.dsde.workbench.leonardo.model.google.ClusterName
+import org.broadinstitute.dsde.workbench.model.{TraceId, UserInfo}
+import org.broadinstitute.dsde.workbench.model.google.GoogleProject
+
+import scala.concurrent.ExecutionContext
+
+class RuntimeRoutes(runtimeService: RuntimeService[IO])(implicit timer: Timer[IO], ec: ExecutionContext) extends UserInfoDirectives {
+  def runtimeRoutes: server.Route = requireUserInfo { userInfo =>
+    implicit val traceId = ApplicativeAsk.const[IO, TraceId](TraceId(UUID.randomUUID()))
+    pathPrefix("google/v1/runtime") {
```

Comment:
We should have equivalent paths in the new scheme for get runtimes, list runtimes, stop/start runtime, and delete runtime. Could be done in a follow-up PR if this one is just scoped to `create runtime`.

Reply:
yea, I'm thinking this PR will be just create runtime since it's the most complicated one..I may add `get runtime` if I need it in test. But I was planning to add the other ones in a separate one